datamodel/samples.d/generate_timestampdefinitions.py

```diff
@@ -202,11 +202,23 @@ def declare_timestamps():
         'BRTH',
         'ALGS',
         "Start Cargo Operations And Services",
-        'jit1_0',
+        'jit1_2',
         'VRDY',
         need_vessel_position=False,
         need_event_location=True,
     )
+    generate_special_timestamp(
+        'Vessel Readiness for Cargo operations SAFE',
```

Comment:
The name will be visible in the UI exactly as it is shown, so the user will see `Vessel Readiness for Cargo operations` + `Vessel Readiness for Cargo operations SAFE`.  As I user without understanding the underlying details, I know which one I would pick - the `SAFE` one. It must be important with the all caps!

So I think we should probably use a different way of telling them apart naming-wise. Maybe something like `Vessel Readiness for Cargo operations [jit1.1]` vs. `Vessel Readiness for Cargo operations [jit1.2+]`, which is not perfect but hopefully signals it is version related thing rather than anything else.